### ISSUE TITLE
Error When Building App: no such file or directory: 'pie'

### ISSUE THREAD

**Issue by rileydnorris** [fastlane/fastlane#14356]
### Question Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Question Subject

I am trying to archive and upload a build to TestFlight but I am running into issues during the building stage despite being able to run the app through Xcode.

#### Question Description

When running `fastlane beta` using these commands:

```
platform :ios do
  desc "Push a new beta build to TestFlight"
  lane :beta do
    sync_code_signing(type: "development") 
    increment_build_number(
    	build_number: latest_testflight_build_number + 1,
    	xcodeproj: "Sympliact.xcodeproj"
     )
    build_app(workspace: "AppName.xcworkspace", scheme: "AppName")
    upload_to_testflight
  end
end
```

I get the following error during the build_app phase:

```
clang: error: no such file or directory: 'pie'
```

It prints out the entire command it runs, which I've run separately and run into the same issue. However I ran the command again removing the word `pie` from the following line and received no error:

```
... -fstack-protector-all pie -MMD -MT dependencies -MF ...
```

Is there a way that I can remove this when running the fastlane command? I've tried researching everywhere I can to no avail, I can't even figure out what the `pie` command is supposed to do. Any help would be much appreciated.

**Comment by janpio** [comment]
Please post the command you are running and the complete output you are getting. Right now we have no context where exactly what is happening.

**Comment by rileydnorris** [comment]
@janpio I believe this is the entire command, sorry I didn't want to post it in case it held any sensitive info but after looking at it that doesn't seem to be the case.

```
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x c -arch armv7 -fmessage-length=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -std=gnu99 -fmodules -gmodules -fmodules-cache-path=/Users/username/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -fmodules-prune-interval=86400 -fmodules-prune-after=345600 -fbuild-session-file=/Users/username/Library/Developer/Xcode/DerivedData/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror=non-modular-include-in-framework-module -Wno-trigraphs -fpascal-strings -Os -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror=return-type -Wdocumentation -Wunreachable-code -Werror=deprecated-objc-isa-usage -Werror=objc-root-class -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -DCOCOAPODS=1 -DCVLIB_IMG_NOCODEC -DGPB_USE_PROTOBUF_FRAMEWORK_IMPORTS=1 -DPB_FIELD_32BIT=1 -DPB_NO_PACKED_STRUCTS=1 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS12.1.sdk -fstrict-aliasing -Wdeprecated-declarations -miphoneos-version-min=10.0 -g -fvisibility=hidden -Wno-sign-conversion -Winfinite-recursion -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wno-semicolon-before-method-body -fembed-bitcode -iquote /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/AppName-generated-files.hmap -I/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/AppName-own-target-headers.hmap -I/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/AppName-all-non-framework-target-headers.hmap -ivfsoverlay /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/all-product-headers.yaml -iquote /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/AppName-project-headers.hmap -I/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/include -I/Users/username/Documents/AppName/ios/AppName/Pods/Firebase/CoreOnly/Sources -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public/AcuantMobileSDK -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public/Firebase -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public/FirebaseAnalytics -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public/FirebaseInstanceID -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public/GoogleAppMeasurement -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public/PPpdf417 -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public/SwiftLint -I/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/DerivedSources/armv7 -I/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/DerivedSources -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos -FPods/AcuantMobileSDK/AcuantMobileSDK.embeddedframework -FPods/AcuantMobileSDK/AcuantMobileSDK.embeddedframework/Resources -FPods/AcuantMobileSDK/AcuantMobileSDK.embeddedframework/Resources/Frameworks -FPods/AcuantMobileSDK/AcuantMobileSDK.embeddedframework/Resources/MicroBlink.bundle -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/AWSCore -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/AWSS3 -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/Auth0 -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/FirebaseCore -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/FirebaseMessaging -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/GoogleUtilities -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/Protobuf -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/SimpleKeychain -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/nanopb -F/Users/username/Documents/AppName/ios/AppName/Pods/FirebaseAnalytics/Frameworks -F/Users/username/Documents/AppName/ios/AppName/Pods/FirebaseInstanceID/Frameworks -F/Users/username/Documents/AppName/ios/AppName/Pods/GoogleAppMeasurement/Frameworks -F/Users/username/Documents/AppName/ios/AppName/Pods/PPpdf417 -F/Users/username/Documents/AppName/ios/AppName/Carthage/Build/iOS -fstack-protector-all pie -MMD -MT dependencies -MF /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/Objects-normal/armv7/AppName_vers.d —serialize-diagnostics /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/Objects-normal/armv7/AppName_vers.dia -c /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/DerivedSources/AppName_vers.c -o /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/Objects-normal/armv7/AppName_vers.o
```

This happens after the `Running script '[CP] Check Pods Manifest.lock'` line and it lets me know of any warnings I have in the code, the entirety of the output is:

```
** ARCHIVE FAILED **


The following build commands failed:
CompileC /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/Objects-normal/armv7/AppName_vers.o /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/DerivedSources/AppName_vers.c normal armv7 c com.apple.compilers.llvm.clang.1_0.compiler
CompileC /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/Objects-normal/arm64/AppName_vers.o /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/DerivedSources/AppName_vers.c normal arm64 c com.apple.compilers.llvm.clang.1_0.compiler
(2 failures)
▸ Compiling AppName_vers.c
▸ Compiling AppName_vers.c

❌  clang: error: no such file or directory: ‘pie’

[21:18:04]: Exit status: 65

+-------------+-------------------------+
|           Build environment           |
+-------------+-------------------------+
| xcode_path  | /Applications/Xcode.app |
| gym_version | 2.117.1                 |
| sdk         | iPhoneOS12.1.sdk        |
+-------------+-------------------------+

[21:18:04]: ▸ CompileC /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/Objects-normal/arm64/AppName_vers.o /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/DerivedSources/AppName_vers.c normal arm64 c com.apple.compilers.llvm.clang.1_0.compiler (in target: AppName)
[21:18:04]: ▸     cd /Users/username/Documents/AppName/ios/AppName
[21:18:04]: ▸     export LANG=en_US.US-ASCII
[21:18:04]: ▸     /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x c -arch arm64 -fmessage-length=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -std=gnu99 -fmodules -gmodules -fmodules-cache-path=/Users/username/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -fmodules-prune-interval=86400 -fmodules-prune-after=345600 -fbuild-session-file=/Users/username/Library/Developer/Xcode/DerivedData/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror=non-modular-include-in-framework-module -Wno-trigraphs -fpascal-strings -Os -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror=return-type -Wdocumentation -Wunreachable-code -Werror=deprecated-objc-isa-usage -Werror=objc-root-class -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -DCOCOAPODS=1 -DCVLIB_IMG_NOCODEC -DGPB_USE_PROTOBUF_FRAMEWORK_IMPORTS=1 -DPB_FIELD_32BIT=1 -DPB_NO_PACKED_STRUCTS=1 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS12.1.sdk -fstrict-aliasing -Wdeprecated-declarations -miphoneos-version-min=10.0 -g -fvisibility=hidden -Wno-sign-conversion -Winfinite-recursion -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wno-semicolon-before-method-body -fembed-bitcode -iquote /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/AppName-generated-files.hmap -I/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/AppName-own-target-headers.hmap -I/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/AppName-all-non-framework-target-headers.hmap -ivfsoverlay /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/all-product-headers.yaml -iquote /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/AppName-project-headers.hmap -I/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/include -I/Users/username/Documents/AppName/ios/AppName/Pods/Firebase/CoreOnly/Sources -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public/AcuantMobileSDK -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public/Firebase -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public/FirebaseAnalytics -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public/FirebaseInstanceID -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public/GoogleAppMeasurement -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public/PPpdf417 -I/Users/username/Documents/AppName/ios/AppName/Pods/Headers/Public/SwiftLint -I/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/DerivedSources/arm64 -I/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/DerivedSources -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos -FPods/AcuantMobileSDK/AcuantMobileSDK.embeddedframework -FPods/AcuantMobileSDK/AcuantMobileSDK.embeddedframework/Resources -FPods/AcuantMobileSDK/AcuantMobileSDK.embeddedframework/Resources/Frameworks -FPods/AcuantMobileSDK/AcuantMobileSDK.embeddedframework/Resources/MicroBlink.bundle -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/AWSCore -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/AWSS3 -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/Auth0 -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/FirebaseCore -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/FirebaseMessaging -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/GoogleUtilities -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/Protobuf -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/SimpleKeychain -F/Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/BuildProductsPath/Release-iphoneos/nanopb -F/Users/username/Documents/AppName/ios/AppName/Pods/FirebaseAnalytics/Frameworks -F/Users/username/Documents/AppName/ios/AppName/Pods/FirebaseInstanceID/Frameworks -F/Users/username/Documents/AppName/ios/AppName/Pods/GoogleAppMeasurement/Frameworks -F/Users/username/Documents/AppName/ios/AppName/Pods/PPpdf417 -F/Users/username/Documents/AppName/ios/AppName/Carthage/Build/iOS -fstack-protector-all pie -MMD -MT dependencies -MF /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/Objects-normal/arm64/AppName_vers.d —serialize-diagnostics /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/Objects-normal/arm64/AppName_vers.dia -c /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/DerivedSources/AppName_vers.c -o /Users/username/Library/Developer/Xcode/DerivedData/AppName-dwufrnvvzhfpqrgdkzsewxwcorlm/Build/Intermediates.noindex/ArchiveIntermediates/AppName/IntermediateBuildFilesPath/AppName.build/Release-iphoneos/AppName.build/Objects-normal/arm64/AppName_vers.o
[21:18:04]: ▸ clang: error: no such file or directory: ‘pie’
```

**Comment by janpio** [comment]
So the `build_app` step is failing for you. But the command there doesn't look like one of the ones generated by fastlane, afaik we just trigger `xcodebuild` which then does the rest via the project configuration it is running.

Can you look through the start of `build_app` action output and see if the command it triggers itself makes sense, before all the Xcode output mumbo jumbo starts?

**Comment by rileydnorris** [comment]
After much configuration and testing I have finally figured it out! I'm not sure if Fastlane was involved, but I've never had this issue before the process of setting it up so maybe it is. Regardless, in case anyone else discovers this issue, the problem was that under:

`Apple Clang - Custom Compiler Flags`

in the build settings for the project, there was a mismatch between debug and release under the `Other C Flags` group. Release's setting was just `-fstack-protector-all pie`, so I just added a `-` to the left of `pie` to make it:

`-fstack-protector-all -pie`

I have no idea how that could have been changed as I haven't messed with any of that, but now it's fixed so it's all good. Thanks for your help!
